### DISCUSSION TITLE
$wire property to v3

### DISCUSSION THREAD

**Discussion by NkrumahAlipio** [alpinejs/alpine#1824] [3. Feature ideas]
Hi, I would like to know why the $wire property was not added to work with livewire

**Comment by Javierko** [comment]
I have used `this.$wire` in my js component.

Check [https://laravel-livewire.com/docs/2.x/alpine-js](url).

**Comment by SimoTod** [comment]
$wire is added by livewire. It's compatible with Alpine v3 but you need to upgrade livewire
